Example/Unit Tests/AnimationCurveSnapshotTests.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  Copyright 2020 Square Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Stagehand
+import StagehandTesting
+import FBSnapshotTestCase
+
+final class AnimationCurveSnapshotTests: FBSnapshotTestCase {
+
+    // MARK: - FBSnapshotTestCase
+
+    override func setUp() {
+        super.setUp()
+        fileNameOptions = [.OS, .screenSize, .screenScale]
+        recordMode = false
+    }
+
+    // MARK: - Tests
+
+    func testLinear() {
+        verifyGraphView(for: LinearAnimationCurve())
+    }
+
+    func testParabolicEaseIn() {
+        verifyGraphView(for: ParabolicEaseInAnimationCurve())
+    }
+
+    func testParabolicEaseOut() {
+        verifyGraphView(for: ParabolicEaseOutAnimationCurve())
+    }
+
+    func testSinusoidalEaseInEaseOut() {
+        verifyGraphView(for: SinusoidalEaseInEaseOutAnimationCurve())
+    }
+
+    func testCubicBezierEaseIn() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
+    }
+
+    func testCubicBezierEaseOut() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
+    }
+
+    func testCubicBezierEaseInEaseOut() {
+        verifyGraphView(for: CubicBezierAnimationCurve.easeIn)
+    }
+
+    func testCubicBezierOvershoot() {
+        verifyGraphView(for: CubicBezierAnimationCurve(controlPoints: (0.5, 0.0), (0.5, 1.3)))
+    }
+
+    // MARK: - Private Methods
+
+    private func verifyGraphView(for curve: AnimationCurve) {
+        let graphSize: CGFloat = 200
+        let margin: CGFloat = 20
+        let containerSize = graphSize + margin * 2
+        let containerView: UIView = .init(frame: .init(x: 0, y: 0, width: containerSize, height: containerSize))
+        containerView.backgroundColor = .white
+
+        let graphView: UIView = .init(frame: .init(x: margin, y: margin, width: graphSize, height: graphSize))
+        let gridLayer = makeGridLayer(frame: graphView.bounds)
+        graphView.layer.addSublayer(gridLayer)
+        let curveLayer = makeCurveLayer(frame: graphView.bounds, curve: curve)
+        graphView.layer.addSublayer(curveLayer)
+        containerView.addSubview(graphView)
+
+        FBSnapshotVerifyView(containerView)
```

Comment:
Let's pass the file and line through here so any test failures show up on the right line.